diff --git a/node_modules/bridge-pca/src/bridge-pca.js b/node_modules/bridge-pca/src/bridge-pca.js
index b53e076..4edc73e 100644
--- a/node_modules/bridge-pca/src/bridge-pca.js
+++ b/node_modules/bridge-pca/src/bridge-pca.js
@@ -130,7 +130,7 @@
 /////  BEGIN BPCA  0.1.6 4g BLOCK       \//////////////////////////////////////
 ////                                     \////////////////////////////////////
 
-import { colorParsley, colorToHex, colorToRGB } from '../node_modules/colorparsley/src/colorparsley.js';
+import { colorParsley, colorToHex, colorToRGB } from 'colorparsley';
 
 
 
